testing/dart/painting_test.dart

```diff
@@ -35,20 +67,24 @@ void main() {
       Vertices.raw(
         VertexMode.triangles,
         Float32List.fromList(const <double>[0.0]),
-      );
-      throw 'Vertices.raw did not throw the expected error.';
-    } on ArgumentError catch (e) {
-      expect('$e', 'Invalid argument(s): "positions" must have an even number of entries (each coordinate is an x,y pair).');
+      ).dispose();
```

Comment:
Can you also add something in here that tries to actually use these objects on a Canvas when assertions are not enabled?

We want to make sure that these invalid objects won't cause a crash.